test/app-for-e2e/pkg/crypto/encrypt.go

```diff
@@ -0,0 +1,71 @@
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"errors"
+	"io"
+)
+
+// SecureCryptoKey represents a super secure key for encryption
+var SecureCryptoKey = NewEncryptionKey()
```

Comment:
this api is only used for integration tests so this works for now. The "issue" is if an entry is added to DB, user restarts server, then tries to query for said entry then the key would be different and decryption would fail.